pkg/virt-launcher/virtwrap/statsconv/converter_test.go

```diff
@@ -52,12 +52,13 @@ var _ = Describe("StatsConverter", func() {
 			in := &libvirt.DomainStats{}
 			inMem := []libvirt.DomainMemoryStat{}
 			devAliasMap := make(map[string]string)
+			inJobInfo := &stats.DomainJobInfo{}
```

Comment:
I would suggest to have at least one non-zero field here, e.g. `inJobInfo.MemDirtyRate = 123`

Reply:
sure.